deep_quoridor/src/train_dqn.py

```diff
@@ -73,8 +79,13 @@ def train_dqn(
                 # Execute action
                 game.step(action)
 
-                # Get new state, reward, etc.
-                next_observation, reward, termination, truncation, _ = game.last()
+                # Get the observation and rewards for THIS agent (not the opponent)
```

Comment:
@alejandromarcu this part is key, otherwise it was getting the rewards for the opponent instead